deploy-to-aks/README.md

```diff
@@ -0,0 +1,41 @@
+# Deploy to AKS
+
+Deploy a docker image by sha to a specific service environment.
+Optionally after deployment
+- run a seed command for review apps
+  - if enabled, it will run 'make review seed-review-app' so this must exist in the Makefile
+- run a smoktest after deployment
+
+If using Google Cloud then GCP_PROJECT_ID abd GCP_WIP variables must be set in the service Makefile.
+
+## Inputs
+- `azure-credentials`: A JSON string containing service principle credentials (Required)
+- `gcp-wip`: The full identifier of the GCP Workload Identity Provider (Optional)
```

Comment:
Link to https://github.com/DFE-Digital/terraform-modules/blob/main/aks/dfe_analytics/README.md#authentication---github-actions

Reply:
added link